unity/PingPongGameExample/PingPongGame/Assets/Scripts/GameManager.cs

```diff
@@ -140,7 +140,30 @@ async Task Start()
       roomIdInput = GameObject.Find("InputFieldRoomId").GetComponent<InputField>();
       roomIdInput.onEndEdit.AddListener(ConnectToServerWithRoomId);
 
+      yield return CheckLocationIsRunning();
+    }
+
+    IEnumerator CheckLocationIsRunning()
+    {
       MobiledgeX.LocationService.ensurePermissions();
+      StartCoroutine(MobiledgeX.LocationService.InitalizeLocationService());
+
+#if UNITY_EDITOR
+      if (Input.location.status == LocationServiceStatus.Failed)
+      {
+        clog("Location service failed for some reason");
```

Comment:
Can we update this message to say something like "Location service is disabled in Unity Editor"

Reply:
I can change this to something like "Location service failed". But actually in Editor, Location Service is not disabled (LocationServiceStatus.Stopped). LocationServiceStatus.Failed would be a critical error

Reply:
sure. mainly I would want to add the fact the error is being printed from the editor path. so "Location service failed in Unity Editor"